lib/pymedphys/docs/README.rst

```diff
@@ -66,86 +55,45 @@ any code that was using the old version of pymedphys. We try to be abreast of
 within the `Release Notes`_.
 
+Mailing list
+**************
 
+A great way to get started with PyMedPhys is to sign up to `the mailing list`_
+and introducing yourself! Tell us a bit about where you're from, what you're
+doing, and what brings you to PyMedPhys!
 
-Documentation
--------------
-
-For App Users
-.............
-
-Documentation of the PyMedPhys application's graphical user interface doesn't
-yet exist. Watch this space.
-
-For Python Users
-................
-
-- Absolute Beginners Tutorial **[Doesn't yet exist]**
-
-  - Start here if you are new to Python and would like an in-depth instruction on
-    how to get started.
-
-- `Quickstart How-To Guide`_
-
-  - Start here if you are fluent in Python and just want a quick instruction on how
-    to get started.
-
-- `Tutorials`_
-
-  - A range of lessons that take you through various complete projects with the
-    intent to be able to show you what can be achieved with writing PyMedPhys based
-    Python software yourself.
-  - These are aimed at you if you don't yet know what you can achieve with
-    PyMedPhys but you would like to learn.
-
-- `How-To Guides`_
-
-  - Guides and recipes for common problems and tasks. These are aimed for you if
-    you already know how to use Python and PyMedPhys and are looking for direction
-    on a specific task you are trying to solve.
-
-- `Reference`_
-
-  - Technical reference for the `library`_ (modules, functions and classes),
-    as well as the available `command line tools`_. This is where you will find
-    listed information for the exposed functionality of PyMedPhys.
-  - This is aimed at you if you know exactly the feature you would like to use, you
-    just want to see what inputs it requires and what outputs it gives.
-
-- `Background`_
-
-  - Explanation and discussion of key topics and concepts. This is aimed at you if
-    you are looking to be able to think about PyMedPhys and its implementations at
-    a higher level and understand more about them.
-
-- Glossary **[Doesn't yet exist]**
-
-  - List of the most important terms
+.. _`the mailing list`: https://groups.google.com/g/pymedphys
 
+|
+|
 
-For Contributors
-................
+Documentation
+==================
 
-- `Getting Started Contributing`_ **[Under construction]**
+The PyMedPhys documentation is split into four categories:
 
-  - Start here for learning how to contribute to PyMedPhys.
+`1. App Users Guide`_: for those who only wish to use ready-made PyMedPhys
+tools
 
-- Developer Guide **[Doesn't yet exist]**
-- `Documentation Guide`_
+`2. Library Users Guide`_: for those building their own Python apps who wish to
+incorporate elements of the PyMedPhys library
 
-  - How to contribute to this documentation.
+`3. Contributors Guide`_: for those who wish to make new contributions to
+either the PyMedPhys library or the PyMedPhys app.
 
-- Maintainers Guide **[Doesn't yet exist]**
+`4. General`_: Material that may apply to any visitor to PyMedPhys.
```

Comment:
This here looks quite odd when I build it:

![image](https://user-images.githubusercontent.com/6559099/105945263-0db34200-60b9-11eb-86f5-af8815cd8824.png)

Reply:
Hmm, I'm not sure why that's happening... maybe ReST doesn't like number points being included in the link? Will try to fix.